pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -228,6 +234,11 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 									"/bin/etcd-druid",
 									"--enable-leader-election=true",
 									"--ignore-operation-annotation=false",
+									"--disable-lease-cache=true",
+									"--enable-backup-compaction=true",
+									"--compaction-workers=3",
+									"--etcd-events-threshold=1000000",
+									"--active-deadline-duration=3h",
```

Comment:
Shouldn't this stuff respect the setting of the gardenlet configuration? Why is this hard-coded here?

Reply:
Right. Will change that

Reply:
Working on this

Reply:
Done